style.css

```diff
@@ -0,0 +1,1125 @@
+@media only screen and (max-width: 768px) {
+  body {
+    margin: 0;
+  }
+
+  header {
+    background-color: black;
+  }
+
+  .home {
+    background-image: url("images/group.svg");
```

Comment:
- [ ] Please I noticed that this background image is missing for the desktop version. Kindly add it as well for the desktop version. 👍